From 24d492c5a0047453fde06cda782c777ff928c6c8 Mon Sep 17 00:00:00 2001
From: Himanshu Nayak <himanshu.nayak@amd.corp-partner.google.com>
Date: Mon, 28 Nov 2022 21:30:08 -0800
Subject: [PATCH] test/test_va_api_query_config: Changed default initialization
 value to VAProfileNone

In Vaapi unit test VAAPIQueryConfig, CheckEntrypointsForProfile
the default initialization value chnaged from zero to VAProfileNone
as zero is a valid profile matches with MPEG2Simple and this unit
test misinterpret it as part of return value from vaQueryConfigProfiles

Signed-off-by: Himanshu Nayak <himanshu.nayak@amd.corp-partner.google.com>
---
 test/test_va_api_query_config.cpp | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/test/test_va_api_query_config.cpp b/test/test_va_api_query_config.cpp
index b5d86db..39a8d1a 100644
--- a/test/test_va_api_query_config.cpp
+++ b/test/test_va_api_query_config.cpp
@@ -42,7 +42,7 @@ TEST_P(VAAPIQueryConfig, CheckEntrypointsForProfile)
     EXPECT_TRUE(maxProfiles > 0)
             << maxProfiles << " profiles are reported";
 
-    Profiles profiles(maxProfiles);
+    Profiles profiles(maxProfiles, VAProfileNone);
 
     EXPECT_STATUS(
         vaQueryConfigProfiles(m_vaDisplay, profiles.data(), &numProfiles));
-- 
2.25.1

